{
    "data": [
        {
            "test_name": "APITest_ExperimentInfoTest",
            "latest_failed_build": "4384",
            "agent_name": "Linux",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "4387",
                    "4386",
                    "4385",
                    "4384",
                    "4383",
                    "4382",
                    "4381",
                    "4380",
                    "4379",
                    "4378",
                    "4377",
                    "4376",
                    "4375",
                    "4374",
                    "4373",
                    "4372",
                    "4371",
                    "4370",
                    "4369",
                    "4368",
                    "4367",
                    "4366",
                    "4365",
                    "4364",
                    "4363",
                    "4362",
                    "4361",
                    "4360",
                    "4359",
                    "4358"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "APITest_ExperimentInfoTest",
                "build": "4384",
                "agent_name": "Linux",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "1.40",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.7.20230703.1019.dev22\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 42 tests...........................Instrument-[Warning] Saving Instrument with no XML data. If this was instrument data you may not be able to load this data back into Mantid, for fitted/analysed data this warning can be ignored.\n\nIn ExperimentInfoTest::test_nexus:\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/API/test/ExperimentInfoTest.h:369: Error: Test failed: Unable to parse File: in &quot;/home/abc/.mantid/instrument/geometryCache/GEM79cd9448aebda416934e7eaafbc9acde3319724b.vtp&quot;\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/API/test/ExperimentInfoTest.h:369: Error: Expected (ws2.loadExperimentInfoNexus(filename, th.file.get(), parameterStr)) not to throw, but it did\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/API/test/ExperimentInfoTest.h:371: Error: Expected (inst-&gt;getName() == &quot;GEM&quot;), found (&quot;&quot; != GEM)\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/API/test/ExperimentInfoTest.h:372: Error: Assertion failed: inst-&gt;getFilename().find(&quot;GEM_Definition.xml&quot;, 0) != std::string::npos\nInstrument-[Warning] Saving Instrument with no XML data. If this was instrument data you may not be able to load this data back into Mantid, for fitted/analysed data this warning can be ignored.\n.Instrument-[Warning] Saving Instrument with no XML data. If this was instrument data you may not be able to load this data back into Mantid, for fitted/analysed data this warning can be ignored.\n.............\nFailed 1 of 42 tests\nSuccess rate: 97%\n\n          Start  495: APITest_ExperimentInfoTest\n"
            }
        },
        {
            "test_name": "MantidQtInterfacesIndirectTestQt5_IndirectFitDataModelTest",
            "latest_failed_build": "4380",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "4387",
                    "4386",
                    "4385",
                    "4384",
                    "4383",
                    "4382",
                    "4381",
                    "4380",
                    "4379",
                    "4378",
                    "4377",
                    "4376",
                    "4375",
                    "4374",
                    "4373",
                    "4372",
                    "4371",
                    "4370",
                    "4369",
                    "4368",
                    "4367",
                    "4366",
                    "4365",
                    "4364",
                    "4363",
                    "4362",
                    "4361",
                    "4360",
                    "4359",
                    "4358"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "MantidQtInterfacesIndirectTestQt5_IndirectFitDataModelTest",
                "build": "4380",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.02",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.7.20230629.1209.dev23\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 25 testsQStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\n\nIn IndirectFitDataModelTest::test_setResolutin_bad_data:\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/scientific_interfaces/Indirect/test/IndirectFitDataModelTest.h:62: Error: Test failed: Workspace2D::getSpectrum, histogram number 1 out of range 1\n...........Instrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\n............\nFailed 1 of 25 tests\nSuccess rate: 96%\n\n          Start 1172: GeometryTest_RectangularDetectorTest\n"
            }
        },
        {
            "test_name": "python.IsisPowder.ISISPowderAbsorptionTest.ISISPowderAbsorptionTest",
            "latest_failed_build": "4378",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "4387",
                    "4386",
                    "4385",
                    "4384",
                    "4383",
                    "4382",
                    "4381",
                    "4380",
                    "4379",
                    "4378",
                    "4377",
                    "4376",
                    "4375",
                    "4374",
                    "4373",
                    "4372",
                    "4371",
                    "4370",
                    "4369",
                    "4368",
                    "4367",
                    "4366",
                    "4365",
                    "4364",
                    "4363",
                    "4362",
                    "4361",
                    "4360",
                    "4359",
                    "4358"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.IsisPowder.ISISPowderAbsorptionTest.ISISPowderAbsorptionTest",
                "build": "4378",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "10.72",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.7.20230629.1209.dev8\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.17 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nE\n======================================================================\nERROR: test_sample_is_set_correctly (ISISPowderAbsorptionTest.ISISPowderAbsorptionTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/Diffraction/isis_powder/test/ISISPowderAbsorptionTest.py&quot;, line 19, in test_sample_is_set_correctly\n    ws = absorb_corrections.run_cylinder_absorb_corrections(\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/Diffraction/isis_powder/routines/absorb_corrections.py&quot;, line 66, in run_cylinder_absorb_corrections\n    ws_to_correct = _calculate__cylinder_absorb_corrections(\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/Diffraction/isis_powder/routines/absorb_corrections.py&quot;, line 90, in _calculate__cylinder_absorb_corrections\n    ws_to_correct = _do_cylinder_absorb_corrections(\nTypeError: _do_cylinder_absorb_corrections() got an unexpected keyword argument &#x27;ms_events&#x27;\n\n----------------------------------------------------------------------\nRan 2 tests in 0.171s\n\nFAILED (errors=1)\n\n"
            }
        }
    ]
}